blog/case-studies/trustmesh

```diff
@@ -0,0 +1,113 @@
+---
+
+slug: hcs-21-trustmesh-scend
+title: HCS-21 TrustMesh – Decentralized Trust Signaling with Scend
+authors: [tonycamero]
+tags: [case-study, hcs-21, trustmesh, scend, civic-tech, reputation]
+date: 2025-10-17T09:00:00-05:00
+-------------------------------
+
+# HCS-21 TrustMesh – Decentralized Trust Signaling with Scend
+
+![TrustMesh Logo](./scendcash_logo.jpg)
```

Comment:
@tonycamero It looks like there's a logo here, but its not in the pull request, one option is inscribing the file and linking to the CDN link